easybuild/easyconfigs/g/GROMACS/GROMACS-5.1.4-CrayGNU-2016.11-cuda-8.0-pat-6.4.5.eb

```diff
@@ -0,0 +1,45 @@
+# Contributed by
+# Luca Marsella (CSCS)
+# Guilherme Peretti-Pezzi (CSCS)
+# Victor Holanda Rusu (CSCS)
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC, Ghent University
+# Authors::   Wiktor Jurkowski <wiktor.jurkowski@uni.lu>, Fotis Georgatos <fotis.georgatos@uni.lu>, \
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+
+name = 'GROMACS'
+version = "5.1.4"
+cudaversion = '8.0'
+craypatversion = '6.4.5'
```

Comment:
Please use the version of the modulefile to be loaded as build_dependency.
E.g.:  patversion = ' 645-cuda'
Then you can use this variable in the build_dependency list.